apps/src/componentLibrary/common/constants.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * This file contains constants that are used across the component library
+ */
+
+import {VisualAppearance} from '@cdo/apps/componentLibrary/typography/types';
+
+/**
+ *  This is the map of component size to body text size (visualAppearance)
+ */
+export const componentSizeToBodyTextSizeMap: {[key: string]: VisualAppearance} =
+  {
+    l: 'body-one',
```

Comment:
If we specifically want to use these keys, we can do something like `type componentSize = 'l' | 'm' | 's' | 'xs'` then declare this as `componentSizeToBodyTextSizeMap: {[key in componentSize]: VisualAppearance}`